README.md

```diff
@@ -39,6 +37,85 @@ $ airflow connections add "wherobots_default" \
     --conn-password "$(< api.key)"
 ```
 
+## Usage
+
+### Execute a `Run` on Wherobots Cloud
+
+Wherobots allows users to upload their code (.py, .jar),
```

Comment:
Is there a limit on or list of the type of file extensions that Wherobots Cloud supports? If so, we should include specifically which file formats are supported.

Reply:
We don't have such official document yet